voxeet-uxkit-reactnative.podspec

```diff
@@ -9,14 +9,14 @@ Pod::Spec.new do |spec|
   spec.license = "Dolby Software License Agreement"
   spec.author = "Voxeet"
   spec.homepage = "https://dolby.io"
-  spec.platform = :ios, "11.0"
+  spec.platform = :ios, "12.0"
   spec.swift_version = "5.5.2"
   spec.source = { :git => "https://github.com/voxeet/voxeet-uxkit-reactnative.git", :tag => "v#{spec.version}" }
   spec.source_files  = "ios/**/*.{h,m}"
   spec.framework = "UIKit"
   spec.dependency "React"
-  spec.dependency "VoxeetUXKit", "1.6.1"
-  spec.dependency "VoxeetSDK", "3.3.1"
+  spec.dependency "VoxeetUXKit", "~>1.0"
+  spec.dependency "VoxeetSDK", "~>3.0"
```

Comment:
Do we need this `VoxeetSDK` dependency as it is already a dependency of `VoxeetUXKit`?
https://github.com/voxeet/voxeet-uxkit-ios/blob/master/VoxeetUXKit.podspec#L14

Reply:
You are right, I've just updated the PR